modules/cactus-web/src/Notification/Notification.test.tsx

```diff
@@ -93,4 +93,23 @@ describe('component: Notification', () => {
 
     expect(queryByText('Error Notification')).toBeNull()
   })
+
+  test('Should call the onClose fn after timeout', async () => {
+    const setOpen = jest.fn()
+    const { queryByText } = render(
+      <StyleProvider>
+        <Notification open={true} vertical="top" horizontal="left">
+          <Alert closeTimeout={3} onClose={setOpen}>
+            Notification
+          </Alert>
+        </Notification>
+      </StyleProvider>
+    )
+
+    expect(queryByText('Notification')).toBeInTheDocument()
+
+    await waitFor(() => {
+      expect(setOpen).toHaveBeenCalled()
+    })
```

Comment:
I think you're supposed to return the expect result, like this (though if it's working, maybe I'm wrong?):
```suggestion
    await waitFor(() => expect(setOpen).toHaveBeenCalled())
```